net-dns/dnsdist/dnsdist-9999.ebuild

```diff
@@ -0,0 +1,147 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/PowerDNS/pdns.git"
+
+if [[ ${PV} = 9999 ]]; then
+	GIT_ECLASS="git-r3"
+	EXPERIMENTAL="true"
```

Comment:
You got me here. The EXPERIMENTAL variable isn't used anywhere else. My bad, I took the mesa ebuild as a template. Will remove it.